### PR TITLE
feat(group-attributes): Add migration to backfill group priority in GroupAttributes

### PR DESCRIPTION
**Pull request by snigdhas**
This is adapted from `0641_backfill_group_attributes` to add `Group.priority` and `Group.first_release` to the GroupAttributes table for SaaS, ST, and self hosted customers.

### CODE REVIEW

**Review by wedamija — COMMENTED** [review]
I just want to note that this caused a backlog in processing these updates last time, so please let sns know, and probably get things scaled up beforehand.

I thought I remember us seeing that we needed another column or two in this table, do we want to get them all in and the do this backfill in one go?

**Comment by snigdhas** [comment]
> I thought I remember us seeing that we needed another column or two in this table

Hm, do you know which columns were missing? It makes sense to add them and do the backfill at once if possible

**Comment by snigdhas** [comment]
We've added `priority` and `first_release_id` from the Group table to GroupAttributes. That should be the only columns we're adding as part of this, so we're ready to finalize the backfill. I'll work with sns/ops before running it to ensure we're ready for a potential backlog.

File: tests/sentry/migrations/test_0730_backfill_group_info_to_group_attributes.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        "consistent": True,
+        "tenant_ids": {
+            "referrer": "group_attributes",
+            "organization_id": project.organization_id,
+        },
+    }
+    request = json_to_snql(json_body, "group_attributes")
+    request.validate()
+    identity = lambda x: x
+    resp = _snuba_query(((request, identity, identity), Hub(Hub.current), {}, "test_api"))[0]
```
Comment (wedamija) [inline tests/sentry/migrations/test_0730_backfill_group_info_to_group_attributes.py:30]: Could be worth validating that your new field is present too

**Review by wedamija — APPROVED** [review]
lgtm. Just a reminder to give sns a heads up so that they can scale those consumers, and to keep an eye on the consumer backlog so that we know it's processing quickly enough

**Comment by wedamija** [comment]
> We've added `priority` and `first_release_id` from the Group table to GroupAttributes. That should be the only columns we're adding as part of this, so we're ready to finalize the backfill. I'll work with sns/ops before running it to ensure we're ready for a potential backlog.

I don't see `first_release_id` here?

**Comment by snigdhas** [comment]
ah I think that got lost in one of the many merge/regenerates - thanks!

File: src/sentry/migrations/0732_backfill_group_info_to_group_attributes.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            "assignee_user_id": assignee["user_id"] if assignee else None,
+            "assignee_team_id": assignee["team_id"] if assignee else None,
+            "owner_suspect_commit_user_id": suspect_owner["user_id"] if suspect_owner else None,
+            "owner_ownership_rule_user_id": ownership_owner["user_id"] if ownership_owner else None,
+            "owner_ownership_rule_team_id": ownership_owner["team_id"] if ownership_owner else None,
+            "owner_codeowners_user_id": codeowners_owner["user_id"] if codeowners_owner else None,
+            "owner_codeowners_team_id": codeowners_owner["team_id"] if codeowners_owner else None,
+        }
+        if group_value.first_release_id:
+            snapshot["first_release_id"] = group_value.first_release_id
```
Comment (wedamija) [inline src/sentry/migrations/0732_backfill_group_info_to_group_attributes.py:134]: Any reason to not just always send this? If it's `None` that's fine, we send none for other fields

File: src/sentry/migrations/0732_backfill_group_info_to_group_attributes.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        group["id"]: group
+        for group in Group.objects.filter(id__in=group_ids).values(
+            "id",
+            "project_id",
+            "status",
+            "substatus",
+            "first_seen",
+            "num_comments",
+            "priority",
+            "first_release",
```
Comment (wedamija) [inline src/sentry/migrations/0732_backfill_group_info_to_group_attributes.py:61]: Nit:  You can use `first_release_id` here.

File: src/sentry/migrations/0732_backfill_group_info_to_group_attributes.py [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+                id=group_id,
+                project_id=group_values["project_id"],
+                status=group_values["status"],
+                substatus=group_values["substatus"],
+                first_seen=group_values["first_seen"],
+                num_comments=group_values["num_comments"],
+                priority=group_values["priority"],
+                first_release_id=(
+                    str(group_values["first_release"]) if group_values["first_release"] else None
+                ),
```
Comment (wedamija) [inline src/sentry/migrations/0732_backfill_group_info_to_group_attributes.py:80]: Why do we need to cast to a string here? Don't we just want the id as an int?

Comment (snigdhas) [reply src/sentry/migrations/0732_backfill_group_info_to_group_attributes.py]: The column type was wrong earlier - this should work as an int now

**Pull request merged**